android/templates/module/generated/build.gradle

```diff
@@ -158,12 +164,15 @@ dependencies {
 	def krollAptJarPath = '<%- krollAptJarPath.replace(/\\/g, '\\\\') %>'
 	annotationProcessor files(krollAptJarPath)
 	compileOnly files(krollAptJarPath)
+	kapt files(krollAptJarPath)
```

Comment:
@ypbnv, can you add the following to the module's "build.gradle" file above the `dependencies{}` section please?
```groovy
// Titanium does not support incremental @Kroll annotation processing. Disable it in kotlin to avoid build warnings.
project.ext['kapt.incremental.apt'] = false
```

This gets rid of the following build warning...
```
w: [kapt] Incremental annotation processing requested, but support is disabled because the following processors are not incremental: org.appcelerator.kroll.annotations.generator.KrollJSONGenerator (NON_INCREMENTAL).
```

Alternatively, you can set `kapt.incremental.apt` via the "gradle.properties" file using our "_buildModule.js", but doing it via a "build.gradle" file like I'm doing above works too.

Reply:
Updated